test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -0,0 +1,72 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-gce-node
+spec:
+  selector:
+    matchLabels:
+      app: csi-gce-driver
+  serviceName: csi-gce
+  template:
+    metadata:
+      labels:
+        app: csi-gce-driver
+    spec:
+      serviceAccount: csi-gce-pd
+      containers:
+        - name: csi-driver-registrar
+          imagePullPolicy: Always
+          image: quay.io/k8scsi/driver-registrar:v0.2.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+            - name: KUBE_NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi
+        - name: gce-driver
+          securityContext:
+            privileged: true
+            capabilities:
+              add: ["SYS_ADMIN"]
```

Comment:
Is this capability required?

Reply:
no, fixed

Reply:
needs, privileged: true for bidirectional mount though